docs/2.9.0/docs/canton/usermanual/FAQ.rst

```diff
@@ -163,6 +163,19 @@ system:
 
 .. literalinclude:: /canton/includes/mirrored/community/app/src/test/resources/documentation-snippets/dev-version-support.conf
 
+.. _how-do-i-enable-preview-features:
+
+How do I enable preview features?
+---------------------------------
+For early access to new features, you need to enable preview version support. A preview protocol version allows the use of features with initial availability.
```

Comment:
I have an alternate text below which is trying to entice customers to try out the Beta features.
 
 ----------
 
Yu need to enable "Beta" support for an Initial Access (IA) feature. For example, enabling a beta protocol version.
  
IA features are fully supported but are available only for Daml Enterprise customers and they also require prior DA approval.  Please contact your relationship manager or DA support if you are interested in using an IA feature. 

If you have enabled `dev version support`, you can use the Beta protocol versions without any additional configuration.